tox.ini

```diff
@@ -0,0 +1,29 @@
+[tox]
+minversion = 1.4.2
+envlist = linters
+skipsdist = True
+
+[testenv]
+#deps = -r{toxinidir}/requirements.txt
+#       -r{toxinidir}/test-requirements.txt
+
+[testenv:linters]
+install_command = pip install {opts} {packages}
+deps =
+  flake8
+commands =
+  flake8 plugins {posargs}
+[testenv:venv]
+# This env is used to push new release on Galaxy
+install_command = pip install {opts} {packages}
+deps = ansible
+commands = {posargs}
+
+[flake8]
+# E123, E125 skipped as they are invalid PEP-8.
```

Comment:
Just to confirm, is this because PEP8 recognizes both E123 styles as a valid, but flake8 does not?

Reply:
That's right :)

Reply:
PEP8 does not detect E123, so flake8 skips it.

Reply:
FYI
Other collections tox settings are also skipping E123.

https://github.com/search?q=org%3Aansible-collections+%22invalid+PEP-8%22&type=Code